great-idea/css/index.css

```diff
@@ -49,16 +49,78 @@ table {
 
 /* Set every element's box-sizing to border-box */
 * {
-    box-sizing: border-box;
+    box-sizing:border-box;
 }
 
 html, body {
     height: 100%;
-    font-family: 'Titillium Web', sans-serif;
+	font-family: 'Titillium Web', sans-serif;
+	margin: auto;
+	width: 95%;
+	max-width: 1000px;
 }
 
 h1, h2, h3, h4, h5 {
     font-family: 'Bangers', cursive;
     letter-spacing: 1px;
     margin-bottom: 15px;
 }
+
+.container{
+	width: 90;
+	margin: auto
```

Comment:
Missing semi-colon. Not the only one though, make sure you remember or you may find unexpected behaviour.